### PR TITLE
jarek/8201: update font color

### PR DESCRIPTION
**Pull request by jaroslawmalekcodete**

### CODE REVIEW

File: js/beakerx_tabledisplay/src/tableDisplay/dataGrid/row/RowManager.ts [hunk 1]
```diff
@@ -47,8 +50,39 @@ export default class RowManager {
...
+      for (let i = 0; i < data.length; i++) {
+          let newRows = [];
+          for (let y = 0; y < data[i].length; y++) {
+              let pair = new DataGridCellValue(data[i][y], fontFun(i,y));
+              newRows.push(pair);
+          }
+          newData.push(newRows);
+      }
+      data = newData;
+      return data;
```
Comment (piorek) [inline js/beakerx_tabledisplay/src/tableDisplay/dataGrid/row/RowManager.ts]:
```
return newData;
```

File: js/beakerx_tabledisplay/src/tableDisplay/dataGrid/row/RowManager.ts [hunk 2]
```diff
@@ -47,8 +50,39 @@ export default class RowManager {
...
+  }
+
+  private defineFontFun(store: BeakerXDataStore, data) {
+      let fontColors = selectFontColor(store.state);
+
+      if (fontColors && (fontColors.length == data.length)) {
+          return (row: number, col: number): string => {
+              return fontColors[row][col]
+          };
```
Comment (piorek) [inline js/beakerx_tabledisplay/src/tableDisplay/dataGrid/row/RowManager.ts]:
```
return (row: number, col: number): string => fontColors[row][color];

File: js/beakerx_tabledisplay/src/tableDisplay/dataGrid/row/RowManager.ts [hunk 3]
```diff
@@ -47,8 +50,39 @@ export default class RowManager {
...
+      } else {
+          return (row: number, col: number): string => {
+              return BeakerXThemeHelper.DEFAULT_DATA_FONT_COLOR;
+          };
```
Comment (piorek) [inline js/beakerx_tabledisplay/src/tableDisplay/dataGrid/row/RowManager.ts]:
```
return (row: number, col: number): string => BeakerXThemeHelper.DEFAULT_DATA_FONT_COLOR;
```

File: js/beakerx_tabledisplay/src/tableDisplay/dataGrid/row/RowManager.ts (hunk 1 above)
Comment (jaroslawmalekcodete) [reply js/beakerx_tabledisplay/src/tableDisplay/dataGrid/row/RowManager.ts]: done

File: js/beakerx_tabledisplay/src/tableDisplay/dataGrid/row/RowManager.ts (hunk 3 above)
Comment (jaroslawmalekcodete) [reply js/beakerx_tabledisplay/src/tableDisplay/dataGrid/row/RowManager.ts]: done

File: js/beakerx_tabledisplay/src/tableDisplay/dataGrid/row/RowManager.ts (hunk 2 above)
Comment (jaroslawmalekcodete) [reply js/beakerx_tabledisplay/src/tableDisplay/dataGrid/row/RowManager.ts]: done

**Pull request merged**